reference/channels.v3.yml

```
@@ -223,7 +223,9 @@ paths:
         - Channel Active Theme
       summary: Get a Channel Active Theme
       operationId: getChannelActiveTheme
-      description: Returns active theme details for a specific channel.
+      description: |-
+        Returns active theme details for a specific channel.
+        Not for use when the store has an active Blueprint (legacy) theme.
```

```suggestion
        Returns details of the theme active on the specified channel.
        Does not support active Blueprint (legacy) themes.
```

`Not for use` doesn't make it clear that you'll get an error if Blueprint is active.